src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -478,6 +478,21 @@ FROM OPENJSON(i.Sequences)
             }
         }
 
+        public async Task<RoaptFinancialApplicationsStatusCounts> GetFinancialApplicationsStatusCounts()
+        {
+            // Note: For now it is easier to run all three queries. It may make sense to do something similar to that done with EPAO
```

Comment:
Should be ok for Private Beta, but we might want to look at just getting the counts once more applications start to come through